src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/FormObject.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#nullable enable
+
```

Comment:
It seems FormObject is not used and can be removed in another PR..

Reply:
Used in WebRequestPSCmdlet.Common.cs line 1160. Maybe we could remove it anyway

Reply:
Yes, it is dead code.

Reply:
done